pkg/plugins/golang/v3/scaffolds/internal/templates/main.go

```diff
@@ -217,7 +218,9 @@ func main() {
 {{- if not .ComponentConfig }}
 	var metricsAddr string
 	var enableLeaderElection bool
+	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "probe-bind-address", ":8081", "The address the probe endpoint binds to.")
```

Comment:
To match the config's `.health.healthProbeBindAddress` key
```suggestion
	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
```

Reply:
Done